core/timeline.py

```diff
@@ -19,11 +19,17 @@ def get_objects(child, date):
     instances = DiaperChange.objects.filter(child=child).filter(
         time__range=(min_date, max_date)).order_by('-time')
     for instance in instances:
+        details = ''
+        if instance.wet:
+            details += '💧'
+        if instance.solid:
+            details += '💩'
```

Comment:
Let's stick to words for now. I'm not against using emoji generally to convey information but I'm worried about the usability of this with teeny emoji and no other hinting. E.g. --

![image](https://user-images.githubusercontent.com/10456740/124697506-b604d800-de9b-11eb-9b7f-e40cd525be11.png)

Reply:
I find the emoji easy to help scan, so how about both?

![](https://media.tenor.com/images/b8d4f0cb6249997a6fa2f0277986d1b1/tenor.gif)